### DISCUSSION TITLE
[help] Can't make `fzf-tmux -p` to popup the window it just render the normal `fzf` command

### DISCUSSION THREAD

**Discussion by dvrd** [junegunn/fzf#3260] [Q&A] [answered]
Any recommendations on how to debug this issue? Doesn't seem to be a common thing so I suppose there is something wrong with my configuration. I installed it with `brew` and tried it on `iTerm`, `WezTerm` & the old local terminal so it doesn't seem to be related to where I'm trying to render the popup from. I have updated versions of `fzf` & `tmux` so that shouldn't be the issue. Has anyone come across this problem?

**Comment by junegunn** [comment]
You can try running `bash -x fzf-tmux -p` and examining what's going on under the hood.

**Reply by dvrd** [reply]
Don't really know what I'm looking for though. This is the output of running that. Would you mind providing a little more guidance?
```
++ command -v fzf
+ fzf=/opt/homebrew/bin/fzf
+ [[ -x /opt/homebrew/bin/fzf ]]
+ args=()
+ opt=
+ skip=
+ swap=
+ close=
+ term=
+ [[ -n '' ]]
++ tput lines
+ lines=43
+ [[ -n '' ]]
++ tput cols
+ columns=170
+ [[ 1 -gt 0 ]]
+ arg=-p
+ shift
+ [[ -z '' ]]
+ case "$arg" in
+ [[ -p =~ ^-[pwhxy] ]]
+ [[ '' =~ -E ]]
+ opt=-E
+ [[ 2 -gt 2 ]]
+ [[ '' =~ ^[0-9%,]+$ ]]
+ [[ '' =~ ^[A-Z]$ ]]
+ continue
+ [[ 0 -gt 0 ]]
+ [[ -z '' ]]
+ /opt/homebrew/bin/fzf
+ exit 130
```

**Reply by junegunn** [reply, marked as answer]
It indicates that `$TMUX` variable is undefined. Are you sure you're running it inside a tmux session?

**Reply by dvrd** [reply]
I'm an idiot I was trying to make it work outside of tmux, everything makes sense now. Thanks for the help
